cabal-install/Distribution/Client/Env/Utils.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE RecordWildCards #-}
+module Distribution.Client.Env.Utils
```

Comment:
I was originally thinking that there would be some "environment" related utils shared between CmdEnv and CmdInstall but taking a closer look at the code it appears it can be refactored such that it doesn't need them. Renamed/moved everything into the an `CmdEnv` module group.